applications/nrf_desktop/configuration/common/ble_adv_def.h

```diff
@@ -19,6 +19,11 @@ const struct {} ble_adv_def_include_once;
 
 static const struct bt_data ad_unbonded[] = {
 	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	/* nRF Desktop devices are assumed to have transmit power set to 0 dBm. */
+	BT_DATA_BYTES(BT_DATA_TX_POWER, 0x0),
```

Comment:
If tx power information is static we should derive it from ble configuration (there is a choice there).

Reply:
`CONFIG_CAF_BLE_STATE_TX_PWR` is set by CAF BLE state only for connections when SoftDevice LL is in use (in HCI command in BLE state we use `BT_HCI_VS_LL_HANDLE_TYPE_CONN` and not `BT_HCI_VS_LL_HANDLE_TYPE_ADV`). This is done after connection is established.
Also the SoftDevice LL ignores the TX power Kconfig option provided by Zephyr (it was used only for SW Split LL that is built with Zephyr).

In that case it may be reasonable to assume that it's 0 by default for SoftDevice LL (I have currently no better idea since we cannot modify advertising data in runtime - moving adv data to RAM may cause memory limitations related issues for our small devices). We should use Kconfig value for configurations with Split LL.